### ISSUE TITLE
why does the random sample operation includes a max-pooling?

### ISSUE THREAD

**Issue by a18700** [QingyongHu/RandLA-Net#129]
According to the implementation of random-sampling, we found out that the sampled point features are indeed the max-pooled neighborhood features of each sampled point.

Is this clarified in the paper?

I can't find about this..

>>
pool_features = tf.reduce_max(pool_features, axis=2, keepdims=True)
